### DISCUSSION TITLE
Submission form validation

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3166] [Q&A] [answered]
Hi,
I am working on a page that requires form field validation (see attached screenshot).  Just wondering if we have any interaction design standards around required field errors?  The potential options are:
- Allow the user to enter required fields without disabling the 'Submit' button.  If a user clicks 'Submit', then the non-entered fields would display an error message.
- Disable the 'Submit' button until a user enters the required fields.
Any advice would be helpful thank you!

<img width="412" alt="Screenshot 2023-04-14 at 3 53 37 PM" src="https://user-images.githubusercontent.com/9394413/232142565-3b836b49-c5e3-4ba1-97ce-e4ec4dae94e2.png">

**Comment by serifluous** [comment, marked as answer]
Hey @tonge495 ! Check out this section about validation on our Form component: https://paste.twilio.design/components/form#validation Let us know if you have any more questions.

**Comment by tonge495** [comment]
Thanks Sarah!
